molecule/default/playbook.yml

```diff
@@ -14,3 +14,8 @@
     args:
       executable: /bin/bash
     when: "'test_volume_compute1' not in existing_volumes.stdout"
+
+- name: Converge
+  hosts: infra1
+  roles:
+    - role: molecule-validate-cinder-deploy
```

Comment:
This is GREAT!

Should the existing tasks be moved to the `main/tasks.yml` file as well?